spring-kafka/src/main/java/org/springframework/kafka/listener/DefaultAfterRollbackProcessor.java

```diff
@@ -133,8 +133,9 @@ private void checkConfig() {
 	public void process(List<ConsumerRecord<K, V>> records, Consumer<K, V> consumer,
 			@Nullable MessageListenerContainer container, Exception exception, boolean recoverable, EOSMode eosMode) {
 
-		if (SeekUtils.doSeeks(((List) records), consumer, exception, recoverable,
-				getRecoveryStrategy((List) records, exception), container, this.logger)
+		List records2 = records;
```

Comment:
Why do we need this variable?
It is going to point to the same `List` object anyway, so mutation on it is going to affect all the variables pointing to it.